src/routes/two-factor-authentication/get-keyuri.ts

```diff
@@ -0,0 +1,34 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithoutClientOrSupplier } from '../../comms/controller';
+
+interface Request {
+  query: {
+    password: string;
```

Comment:
Would you mind moving this into the body? It requires changing the method to post, but that's fine I think.